packages/client-sdk-nodejs/src/config/middleware/cache-request-logging.ts

```diff
@@ -387,7 +387,7 @@ const convertSetDifferenceRequest: RequestToLogInterfaceConverterFn<
       ? request.subtrahend.set.elements.map(element =>
           convertBytesToString(element)
         )
-      : undefined,
+      : null,
```

Comment:
🤔  Ok, I need to revisit my thinking. This should 100% be `undefined`, not `null`.

Reply:
lol. :)

Ternary operators won't let me omit the block after the `:`, and I can't use `undefined` there. This class/file isn't public enough to warrant too much churn but if you do have another idea how to solve this particular one lmk.